### DISCUSSION TITLE
about sample of fetch api ，can't connect network on real device

### DISCUSSION THREAD

**Discussion by zileyuan** [zepp-health/zeppos-forum#317] [Q&A] [answered]
use emulator can connect network，and return respone，but zeus preview install on my gtr4，touch the button alway return “error” message， why

**Comment by MyoungXUE** [comment, marked as answer]
Hello, the network request can work on the Simulator, which means there is no problem with the code.

To access the network on the real machine, you need to make sure that the watch and phone are connected via Bluetooth.

If Bluetooth is connected but still unable to request, please follow the prompts below to submit user feedback to us. Thank you for your cooperation.

https://docs.zepp.com/docs/guides/faq/upload-log/

**Comment by zileyuan** [comment]
yes, the zeppApp run on real machine, and connected  via Bluetooth  and mobile has internet, but the miniprogram can't get data

**Reply by MyoungXUE** [reply]
Please tell me your Zepp ID when you have completed your feedback submission.

**Reply by zileyuan** [reply]
where  Submitting feedback in Zepp "Profile" - > "User feedback" can greatly shorten the time to solve problems.

in zeppApp or platform or github ?

zileyuan@gmail.com

**Reply by MyoungXUE** [reply]
In Zepp App, i will supplement this detail in docs. Thank you!

**Reply by zileyuan** [reply]
ID：1076700635

**Reply by zileyuan** [reply]
real machine is gtr4 with zeppos3.0
